src/ulvt/prime_field_sumcheck/utils/interpolate.hpp

```diff
@@ -0,0 +1,9 @@
+#include "../../finite_fields/qm31.cuh"
+
+constexpr QM31 one_half = (uint32_t) 0x40000000;
+
+QM31 interpolate_at(QM31 challenge, QM31 evals[3]) {
```

Comment:
```suggestion
inline QM31 interpolate_at(QM31 challenge, QM31 evals[3]) {
```
to ensure that multiple definitions in different compilation that include this header units do not result in link errors.